projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -2134,11 +2134,14 @@ function wpcom_launchpad_find_site_about_page_id() {
 	$headstart_about_pages = array_filter(
 		$annotation['content'],
 		function ( $page ) {
-			if ( 'page' !== $page['post_type'] ) {
+			if ( isset( $page['post_type'] ) && 'page' !== $page['post_type'] ) {
```

Comment:
I've not tested it but this should also work:
```suggestion
			if ( ( $page['post_type'] ?? null ) !== 'page' ) {
```